autonomy/cli/analyse.py

```diff
@@ -165,37 +168,47 @@ def parse_logs(file: Path) -> None:
         raise click.ClickException(str(e)) from e
 
 
-@abci_group.command(name="check-handlers")
-@click.argument(
-    "packages_dir",
-    type=click.Path(dir_okay=True, exists=True),
-    default=Path.cwd() / "packages",
-)
+@analyse_group.command(name="handlers")
+@pass_ctx
 @click.option(
-    "--skip",
+    "--common-handlers",
+    "-h",
     type=str,
-    default="abstract_abci",
-    help="Specify which skills to skip. Eg. skill_0,skill_1,skill_2",
+    default=[
+        "abci",
+    ],
+    help="Specify which handlers to check. Eg. -h handler_a -h handler_b -h handler_c",
+    multiple=True,
```

Comment:
Utilise multiple values for a click option to take multiple values for common handlers. So

```
autonomy analyse abci check-handlers --common abci,http,contract_api,ledger_api,signing --skip abstract_abci,counter,counter_client,hello_world_abci
```
becomes

```
autonomy analyse handlers -h abci -h http -h contract_api -h ledger_api -h signing -i abstract_abci -i counter -i counter_client -i hello_world_abci
```

Reply:
what is the advantage of this?

Reply:
The only advantage is this is more systematic and reduces chances of the user providing badly formatted input strings